tutorials/examples/train_line.py

```diff
@@ -308,16 +310,31 @@ def train(
         policy_std_max=policy_std_max,
     )
     pb = StepEstimator(environment, pb_module, backward=True)
-    gflownet = TBGFlowNet(pf=pf, pb=pb, logZ=0.0)
+    gflownet = TBGFlowNet(pf=pf, pb=pb, logZ=0.0).to(
+        torch.device("cpu")
+    )  # TODO: make configurable.
```

Comment:
can be at constructor now?

Reply:
Also, why do we need this? You added the conftest.py, which should take care of this

Reply:
conftest.py does not cover this file, first of all, and second of all, this file should work outside of pytest.

Reply:
"can be at constructor now?" I'm not sure what you mean by this.

Reply:
I meant during construction, i.e. passing it to init